lib/locales/br/frontend/organizations.json

```
@@ -1,7 +1,6 @@
 {
   "all_organizations_title": "Organizações lançando Hackathons e Challenges",
   "all_organizations_page_description": "Startups, empresas, universidades, investidores, comunidades de tecnologia, escolas de codificação e outras organizações lançando hackathons na plataforma TAIKAI Hackathon",
-  "launch_your_challenge": "Lançar Hackathon",
```

We are using this on `components/products/launch-challenge-modal.js`
```suggestion
  "launch_your_challenge": "Lançar Hackathon",
```